ProcessLib/LIE/SmallDeformation/SmallDeformationProcess.cpp

```diff
@@ -356,6 +356,18 @@ void SmallDeformationProcess<DisplacementDim>::initializeConcreteProcess(
         }
     }
 
+    auto mesh_prop_w_n = MeshLib::getOrCreateMeshProperty<double>(
+        const_cast<MeshLib::Mesh&>(mesh), "w_n",
+        MeshLib::MeshItemType::Cell, 1);
+    mesh_prop_w_n->resize(mesh.getNumberOfElements());
```

Comment:
The resize is not necessary. It's already done in the MeshLib::getOrCreateMeshProperty...